modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm

```diff
@@ -303,10 +303,6 @@ GLOBAL_LIST_EMPTY(customizable_races)
 	default_mutant_bodyparts = list("tail" = "Cat", "ears" = "Cat")
 	learnable_languages = list(/datum/language/common, /datum/language/nekomimetic)
 
-/datum/species/human/monkey
-	mutant_bodyparts = list()
-	default_mutant_bodyparts = list("tail" = "Monkey")
```

Comment:
what is this for?

Reply:
Redundant and overrode human species for some reason